elasticsearch/src/main/java/org/hibernate/search/backend/elasticsearch/impl/ElasticsearchIndexManager.java

```diff
@@ -263,6 +266,14 @@ private void createIndexMappings() {
 		}
 	}
 
+	private String analyzerName(DocumentFieldMetadata fieldMetadata) {
+		AnalyzerReference analyzer = fieldMetadata.getAnalyzer();
```

Comment:
Should we raise an exception if it's an actual Lucene analyzer? I'd say let's at least raise a warning that's going to take no effect.